clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2794,6 +2794,19 @@ def BaseClassAddrOp : CIR_Op<"base_class_addr"> {
 // FuncOp
 //===----------------------------------------------------------------------===//
 
+// The enumeration values are not necessarily in sync with `clang::CallingConv`
+// or `llvm::CallingConv`.
```

Comment:
Parsimony nit/question: Should we even bother with using LLVM's numerical IDs then? The parsing helper creates a maximum-enum-element-sized `SmallVector` every time a CC is parsed, IIUC. On the other hand, this is only invoked if a `cc(...)` is present on the `cir.func`, and the LLVMIR dialect does the same.

Reply:
Indeed! Updated: use natural numbering for enum values of CIR calling convs to avoid confusion.

Reply:
Agreed!